src/tests/builders/dependency.rs

```diff
@@ -15,7 +15,7 @@ impl DependencyBuilder {
             explicit_name_in_toml: None,
             name: name.to_string(),
             registry: None,
-            version_req: u::EncodableCrateVersionReq(semver::VersionReq::parse(">= 0").unwrap()),
+            version_req: u::EncodableCrateVersionReq(semver::VersionReq::parse("> 0").unwrap()),
```

Comment:
Added to the weekly meeting to discuss if disallowing `>= 0` (as we do for the equivalent `*` wildcard) is an acceptable change.